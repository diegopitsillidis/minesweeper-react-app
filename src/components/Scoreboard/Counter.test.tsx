import React from 'react';
import { render } from "@testing-library/react";
import { Counter, CounterProps } from "./Counter";

it('Counter renders correctly', () => {
    const args = {
        children: '010',
    } as CounterProps;

    const { asFragment } = render(<Counter {...args} />);

    expect(asFragment()).toMatchSnapshot();
});