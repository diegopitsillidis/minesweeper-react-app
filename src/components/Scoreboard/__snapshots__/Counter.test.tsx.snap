// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter renders correctly 1`] = `
<DocumentFragment>
  <div
    class="css-10qcrn4"
  >
    010
  </div>
</DocumentFragment>
`;
